src/AvatarMember.jsx

```diff
@@ -21,15 +24,16 @@ class AvatarMember extends React.Component {
 			{
 				'avatar--org': org,
 				'avatar--fbFriend': fbFriend,
+				'avatar--noPhoto': (member.photo || {}).photo_link == undefined
```

Comment:
just use `'avatar--noPhoto': !(member.photo || {}).photo_link` - no need to check that it's a particular kind of falsy.